### PR TITLE
Update build.xml

### PR DESCRIPTION
**Pull request by robinshine**

### CODE REVIEW

File: componentA/build/build.xml [hunk 1]
```diff
@@ -6,12 +6,12 @@
 		<pathelement location="build/classes" />
```
Comment (yjshen324) [inline componentA/build/build.xml]: Use builds

File: componentA/build/build.xml [hunk 2]
```diff
@@ -6,12 +6,12 @@
...
 	</path>			
 	
-	<target name="deploy">
+	<target name="release">
 		<mkdir dir="build/classes"/>
         <javac srcdir="src" destdir="build/classes"/>
```
Comment (yjshen324) [inline componentA/build/build.xml]: use java

File: componentA/build/build.xml (hunk 1 above)
Comment (robinshine) [reply componentA/build/build.xml]: Hmm...

File: componentA/build/build.xml (hunk 2 above)
Comment (robinshine) [reply componentA/build/build.xml]: Hmm

File: componentA/build/build.xml [hunk 3]
```diff
@@ -6,12 +6,12 @@
...
 	</target>
 	
-	<target name="release" depends="compile">
+	<target name="test" depends="compile">
 		<mkdir dir="build/classes"/>
 		<javac srcdir="test" destdir="build/classes" classpathref="classpath.test"/>
```
Comment (robinshine) [inline componentA/build/build.xml]: HAHAHA

File: componentA/build/build.xml [hunk 4]
```diff
@@ -6,12 +6,12 @@
 		<pathelement location="build/classes" />
 	</path>			
 	
-	<target name="deploy">
```
Comment (robinshine) [inline componentA/build/build.xml]: Me too!!

**Comment by robinshine** [comment]
Are there any differences?

File: componentA/build/build.xml (hunk 4 above)
Comment (robinshine) [reply componentA/build/build.xml]: DADADA!!!

**Pull request merged**